### PR TITLE
Display visual feedback after clicking the “Copy” button

### PR DESCRIPTION
**Pull request by yonihemi**
Found it confusing to not have any feedback for Copy, so added a tooltip.

### CODE REVIEW

**Comment by sindresorhus** [comment]
I'm not sure about this. Why would you expect any feedback? You don't get any feedback if you press the "Edit => Copy" menu item in any app.

**Comment by yonihemi** [comment]
You might be right, it could be me getting used to iOS-y feedback style, but there is a subtle indication in the flickering of the Edit menu which doesn't happen here (clicking button or pressing Cmd-C). Feel free to reject if this feels superfluous.

**Comment by sindresorhus** [comment]
@yonihemi I'll need to think about this more.

Note to self: If we decide to go for this, the tooltip needs to automatically disappear after a set amount of seconds, maybe 4 seconds.

File: Gifski/ConversionCompletedViewController.swift [hunk 1]
```diff
@@ -23,6 +23,13 @@ final class ConversionCompletedViewController: NSViewController {
 		maxWidth: 260
 	)
 
+	private lazy var copyTooltip = Tooltip(
+		identifier: "copyCompletedTips",
+		text: "GIF copied to clipboard.",
```
Comment (sindresorhus) [inline Gifski/ConversionCompletedViewController.swift]:
```suggestion
		text: "GIF copied to the clipboard",
```

**Comment by sindresorhus** [comment]
An alternative approach:

**Comment by sunshinejr** [comment]
Ah yeah some feedback on copying (either via button or command + c) would be great. Tooltip might be just a bit too much, but I like the idea of updating the "Copy" button for a second and go back to its original state.

**Comment by yonihemi** [comment]
Agreed, this is more elegant. Changed implementation to the one suggested on the button itself.

**Pull request merged**